.github/workflows/reuseable-snapshot-timestamp.yml

```diff
@@ -107,10 +112,18 @@ jobs:
           sudo apt-get install libpcsclite-dev
           go build -o tuf -tags=pivkey ./cmd/tuf/
 
-      # Snapshot and timestamp
-      - name: snapshot and timestamp
-        if: ${{ inputs.snapshot_timestamp }}
+      # Snapshot
+      - name: snapshot
+        if: ${{ inputs.snapshot }}
+        run: |
+          echo "SNAPSHOT=1" >> $GITHUB_ENV
+          ./scripts/step-3.sh
+
+      # Timestamp
+      - name: timestamp
+        if: ${{ inputs.timestamp }}
```

Comment:
i think you need to run a timestamp if snapshot was true as well

Reply:
Yep, you're right, I was thinking we could separate these two. Lemme update that.